### PR TITLE
classic: if classic fails to get created, undo the bind mounts

### PR DESCRIPTION
**Pull request by mvo5**
The bind mounts need to be undone otherwise the all the stuff that
is bind mounted will get deleted (like the users home dir)

### CODE REVIEW

File: classic/run.go [hunk 1]
```diff
@@ -109,5 +108,14 @@ func Destroy() error {
 		}
 	}
 
+	return nil
+}
+
+// Destroy destroys a classic environment, i.e. unmonts and removes all files
+func Destroy() error {
+	if err := unmountBindMounts(); err != nil {
+		return fmt.Errorf("can not unmount")
```
Comment (chipaca) [inline classic/run.go]: cannot

File: asserts/asserts.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ type AssertionType struct {
 // Understood assertion types.
 var (
 	AccountKeyType = &AssertionType{"account-key", []string{"account-id", "public-key-id"}, assembleAccountKey}
-	// XXX: is series actually part of the primary key?
+	// ModelType XXX: is series actually part of the primary key?
```
Comment (chipaca) [inline asserts/asserts.go]: did this sneak in by accident or on purpose?

**Comment by chipaca** [comment]
:+1:

File: asserts/asserts.go (hunk 2 above)
Comment (mvo5) [inline asserts/asserts.go]: On purpose, vet failed otherwise

Comment (chipaca) [inline asserts/asserts.go]: Don't tell @niemeyer or he'll be after the vet's jugular.

**Pull request merged**